README.md

```diff
@@ -19,7 +19,7 @@ To create a proposal:
 1. Create a new branch off of `main`
 2. Create a document in the `accepted` folder and use [meta/template.md](meta/template.md) as your
    starting point.
-3. Run `./update-index > INDEX.md` to update the [INDEX](INDEX.md).
+3. Run `./update-index > INDEX.md` to update the [INDEX](INDEX.md).  On Windows use `python3 update-index > INDEX.md`
```

Comment:
We could also ask users to use the `python3`-style command everywhere, if that reduces confusion?

Do you think it makes sense to update the CI error message too? https://github.com/dotnet/designs/blob/26606a24513a7872f6adf2479022077edd01c81d/.github/workflows/index.yml#L19

Reply:
Sure, I can do that.